astropy/utils/masked/core.py

```diff
@@ -1017,6 +1017,18 @@ def _masked_result(self, result, mask, out):
         # For inplace, the mask will have been set already.
         return out
 
+    def __array_wrap__(self, obj, context=None, return_scalar=False):
```

Comment:
"For ufuncs, it is hoped to eventually deprecate this method in favour of [__array_ufunc__](https://numpy.org/doc/stable/reference/arrays.classes.html#numpy.class.__array_ufunc__)".
Should a comment or test be added to call attention to this eventual deprecation?

Reply:
Hmm, that numpy comment (which I introduced...) is a little out of date - since it hasn't been deprecated in numpy 2.0, it will probably not happen... Though the error message here is correct, `__array_wrap__` really should not be used for ufuncs any more. Anyway, perhaps best to just leave things as they are here...

Reply:
@nstarman - would you be able to merge? @nstarman approved as well.